SEGSupport/CMakeLists.txt

```diff
@@ -1,29 +1,42 @@
 #-----------------------------------------------------------------------------
-set(MODULE_NAME SEG2NRRD)
+# Not used
+set(MODULE_NAME SEGSupport)
 
 #-----------------------------------------------------------------------------
 set(MODULE_INCLUDE_DIRECTORIES
   )
 
-set(MODULE_SRCS
-  SEG2NRRD.cxx
-  framesorter.h
-  SegmentAttributes.h
-  )
-
+#-----------------------------------------------------------------------------
 set(MODULE_TARGET_LIBRARIES
+  ${DCMTK_LIBRARIES}
   ${ITK_LIBRARIES}
-  )
+  xml2 z
```

Comment:
I had some compile problems, but I think those might have been resolved after DCMTK cmake issues were resolved. Need to test. Steve, did you try building without those?